js/asideShowTimes.js

```diff
@@ -0,0 +1,97 @@
+import { filterMovies } from './filterMovies.js'
+
+async function init() {
+    const database = await filterMovies("showtimes");
+    sortByTime(database);
+}
+
+// This function is coded so that it can take asmany dates as it wants, not just for today and tomorrow, incase it would be needed in the future
+function sortByTime (database) {    
+    // An array for all the different dates from the database (no other data then dates)
+    const dates = [];
+    // An array for all the movies from different dates from the database (all data from database, in different arrays depending on date of showtimes)
+    const days = [];
+
+    // Goes through all showtimes, converts the dates to readable format, if date is uniqe put in dates array
+    database.forEach(data => {
+        let day = new Date(data.day);
+        data.day = (day.toDateString())
+        
+        if(dates.indexOf(data.day) === -1){
+            dates.push(data.day);
+        }
+    });   
```

Comment:
Hot tip! Check out Sets for dealing with unique values in JavaScript, it's super handy! 😁

Reply:
Tnx, I will read up about it!